src/main/java/coresearch/cvurl/io/mapper/CVType.java

```diff
@@ -0,0 +1,32 @@
+package coresearch.cvurl.io.mapper;
+
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+
+/**
+ * Class that describes type with generics. Used as parameter to some of {@link coresearch.cvurl.io.request.Request}
+ * methods to provide ability to parse response to body to class with generics.
+ * Usage example: <br/>
+ * {@code cvurl.get(url).asObject(new CVType<List<Users>() {});} <br/>
+ * this code snippet would parse response body to {@code List<User>}
+ */
+public abstract class CVType<T> {
```

Comment:
Nice, but its a bad naming)

Reply:
What do you think it should be ? My other options are CVTypeReference, BodyTypeReference, BodyType

Reply:
I don't think CV should be mentioned anyhow. Let's use smth like ResolvedType, ParametrizedType etc

Reply:
I prefer BodyType or GenericBodyType or ParametrizedBodyType then. Because ResolvedType tells me nothing really and ParametrizedType is already a class in java.lang.reflect.

Reply:
my favorite is BodyType - it's short and descriptive